src/deluge/gui/menu_item/midi/devices.cpp

```diff
@@ -26,7 +26,7 @@ extern menu_item::Submenu midiDeviceMenu;
 
 namespace menu_item::midi {
 
-Devices devicesMenu{"Devices"};
+extern Devices devicesMenu{"Devices"};
```

Comment:
Could you remove the brace-initializer here so we know it's not getting used? And for the other extern?